### ISSUE TITLE
Refresh Fails 2022 Super Duty

### ISSUE THREAD

**Issue by shackrat** [itchannel/fordpass-ha#223]
The integration sets up okay and some sensors populate when the integration is loaded, but automatic refresh fails consistently with the following errors in the logs:

```
Logger: homeassistant.components.sensor
Source: custom_components/fordpass/sensor.py:351 
Integration: Sensor ([documentation](https://www.home-assistant.io/integrations/sensor), [issues](https://github.com/home-assistant/home-assistant/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+sensor%22)) 
First occurred: 5:28:05 PM (9 occurrences) 
Last logged: 9:00:26 PM

Error adding entities for domain sensor with platform fordpass
Error while setting up fordpass platform for sensor
fordpass: Error on device update!
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 428, in async_add_entities
    await asyncio.gather(*tasks)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 692, in _async_add_entity
    await entity.add_to_platform_finish()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 789, in add_to_platform_finish
    self.async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 545, in async_write_ha_state
    self._async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 586, in _async_write_ha_state
    attr.update(self.extra_state_attributes or {})
  File "/config/custom_components/fordpass/sensor.py", line 401, in extra_state_attributes
    return self.get_value("attribute")
  File "/config/custom_components/fordpass/sensor.py", line 351, in get_value
    zone[key] = value["value"]
TypeError: 'NoneType' object is not subscriptable
```

Thanks!

**Comment by itchannel** [comment]
It appears one of the light switch statuses is returning null for your vehicle, never seen that before. Will add a quick check for a null status in the next version 1.40

**Comment by shackrat** [comment]
I concur..  I modified sensor.py to add the check and the integration is working fine now..

```
                    if (
                        self.coordinator.data[self.sensor]["lightSwitchStatusData"]
                        != None
                    ):
                        for key, value in self.coordinator.data[self.sensor][
                            "lightSwitchStatusData"
                        ].items():
                            if value is not None:
                                zone[key] = value["value"]
```

Thanks again.

**Comment by itchannel** [comment]
Fixed in 1.40
